src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -116,7 +117,16 @@ export class ListOfComplaintsComponent implements OnInit {
         if (complaint.accepted != undefined) {
             this.correctionRound += 1;
         }
-        const url = [`/course-management/${this.courseId}/${exercise.type}-exercises/${exercise.id}/submissions/${submissionId}/assessment`];
+        const url = getLinkToSubmissionAssessment(
+            exercise.type,
+            this.courseId,
+            exercise.id!,
+            studentParticipation.id,
+            submissionId,
+            this.examId ? this.examId : 0,
+            exercise.exerciseGroup?.id!,
+            complaint.result.id,
+        );
```

Comment:
A few attributes that were not needed to be loaded before are added to this call (`studentParticipation.id`, `submissionId`, `complaint.result.id`) - is it ensured that these are always loaded and defined?

Reply:
The first line of this method checks for the existence and definedness of all values that may not be null, so yes :D